<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FactorySetPage</class>
 <widget class="QWidget" name="FactorySetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color: rgb(51,153,255);
border:0px;
font:18px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;
}

QPushButton:disabled
{
outline:none;
border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(180, 180, 180);
border-radius:2px;
}</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnM1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>设置为M1端</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnM2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>设置为M2端</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Restart">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>重启显示屏</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;
color:red;</string>
   </property>
   <property name="text">
    <string>设置完成后，请点击［重启显示屏］按钮，重新启动显示屏。</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
